src/test/java/com/exasol/adapter/dialects/rls/AbstractRowLevelSecurityIT.java

```diff
@@ -88,19 +95,16 @@ private static void createConnectionDefinition() {
 
     private static String getJdbcUrl() {
         final int port = EXASOL.getDefaultInternalDatabasePort();
-        if (exasolVersionSupportsFingerprintInAddress()) {
+        if (exasolVersionSupportsFingerprintInAddress(EXASOL.getDockerImageReference())) {
             final String fingerprint = EXASOL.getTlsCertificateFingerprint().get();
-            return "jdbc:exa:localhost/" + fingerprint + ":" + port;
+            return "jdbc:exa:localhost:" + port + ";validateservercertificate=1;fingerprint=" + fingerprint;
         }
         return "jdbc:exa:localhost:" + port + ";validateservercertificate=0";
     }
 
-    protected static boolean exasolVersionSupportsFingerprintInAddress() {
-        final ExasolDockerImageReference imageReference = EXASOL.getDockerImageReference();
-        return (imageReference.getMajor() >= 7) && (imageReference.getMinor() >= 1);
-    }
 
     // [itest->dsn~query-rewriter-adds-row-filter-for-tenants~1]
+    @Tag("Specific")
```

Comment:
Why did you add this?

Reply:
Well  you weren't supposed to review this yet as it's WIP (Why are you reviewing this already? 😛 )
In this case we had a problem with container cleanup not working between some of the derived test classes.
We added the tag to have a faster test case. It's going to be removed soon :)